File: Drivers/FSCC Drivers/linux/lib/python/examples/registers.py
import fscc

if __name__ == '__main__':
    p = fscc.Port(0)

    p.registers.CCR0 = 0x0011201c
    p.registers.BGR = 10

    ccr1 = p.registers.CCR1
    ccr2 = p.registers.CCR2

File: Drivers/FSCC Drivers/windows/lib/fscc/net/examples/registers.cs
using Fscc;

public class Tutorial
{
    public static int Main(string[] args)
    {
        Fscc.Port p = new Fscc.Port(0);

        p.Registers.CCR0 = 0x0011201c;
        p.Registers.BGR = 0;
        
        var ccr1 = p.Registers.CCR1;
        var ccr2 = p.Registers.CCR2;

        return 0;
    }
}